machine-learning/solutions/seismic_salt_detection/trainer/model.py

```diff
@@ -73,51 +73,40 @@ def create_regressor(config, parameters):
     Returns:
         A configured and ready to use `tf.estimator.DNNRegressor`
     """
-
     # Mean and Standard Deviation Constants for normalization.
-    with file_io.FileIO(parameters.mean_path, mode='r') as f:
-        mean = pickle.load(f)
-    with file_io.FileIO(parameters.std_path, mode='r') as f:
-        std = pickle.load(f)
+    mean = np.float32(parameters.depth_mean)
```

Comment:
That these values need to be hardcoded is weird and seems like an opportunity for error.  Is there another alternative?

Reply:
they are not hardcoded. You can pass them as parameters when running the script (only the defaults are hard coded if parameters are missing when running the script)

Reply:
"hardcoded" was a poor choice of words.  My reading of this is that mean and std_dev are information that's used at training time but need to be explicitly passed here at inference.  If the case, it means that the caller has to remember facts about training.  That seems like an easy place for future bugs to emerge from.